app/flows/marriage_abroad_flow/outcomes/countries/denmark/_ceremony_country.erb

```diff
@@ -1,27 +1,31 @@
+<% # TBC %>
```

Comment:
Can we take these out, from all 3 new ERBs?

Reply:
Yep, I originally left them in to make it obvious that the wrapped content was yet to be confirmed, however the changes have been approved so TBC lines can now be removed. I'll do this now.